src/components/App/MapRenderer/index.tsx

```diff
@@ -0,0 +1,125 @@
+import React, {useEffect, useState} from 'react';
+import { Map, ZoomControl, Marker } from 'pigeon-maps';
+import { maptiler } from 'pigeon-maps/providers';
+import { Geocoder } from '@maptiler/geocoder';
+import mapRendererStyles from './MapRenderer.module.scss';
+
+const MAPTILER_ACCESS_TOKEN = process.env.NEXT_PUBLIC_MAPTILER_ACCESS_TOKEN;
+
+const mapTilerProvider = maptiler(
+  MAPTILER_ACCESS_TOKEN,
+  'outdoor'
+);
+
+type Anchor = [number, number];
+
+type Marker = {
+  anchor: Anchor;
+  zoom: number;
+};
+
+type MapRendererProps = {
+  markers: Marker[];
+};
+
+const MapRenderer = ({ markers }: MapRendererProps) => {
+  const [geocoderSearchValue, setGeocoderSearchValue] = useState<string>('');
+  const [geocoderGeocodedPlace, setGeocoderGeocodedPlace] = useState<string>('');
+  let geocoder = new Geocoder({
+    key: MAPTILER_ACCESS_TOKEN
+  });
+  
+  const [center, setCenter] = useState<Anchor>([
+    49.882114, -119.477829
```

Comment:
I think this is the branch is the one you mentioned still has things to be worked on, so I'll leave the review here for now

Reply:
this is being updated by another branch, I guess we can stale this one